### ISSUE TITLE
Additional currency

### ISSUE THREAD

**Issue by graileddotcom** [graileddotcom/grailed#1]
~~Every time the item is edited it's reverted to dollars~~ FIXED, changed to question/discussion about the feature

**Comment by albo-cadd9** [comment]
Also imo would make more sense having it as a global variable instead of selecting it for every item

**Comment by albo-cadd9** [comment]
@muffs @liamgriffiths So what's the status for this? A buyer just told me he wants to pay in € if possible, but he can't. It's quite annoying since I feel like I'm donating money to paypal :| (I'm in the eurozone btw)

**Comment by julianlconnor** [comment]
@albo-cadd9 This bugfix will go out on Monday ;)

**Comment by graileddotcom** [comment]
nice!

**Comment by albo-cadd9** [comment]
oh sorry I replied by mail with the wrong account. long day...

**Comment by albo-cadd9** [comment]
Oh I see you've implemented it, great! Before closing it, one question: I tried paying for a 300 USD item in EUR, and it converted the amount to 267.59€, which is (according to google) 295.26 USD. So it uses a 0.89 rate (paypal's rate) instead of the real 0.91. So basically if I ask a buyer to pay in dollars or euro it doesn't make any difference to me, how's that? What am I missing?

**Comment by ArunGupta25** [comment]
This is what we use to get our exchange rate: https://openexchangerates.org/. It's not paypal.

**Comment by albo-cadd9** [comment]
Ok, I didn't mean you used paypal's currency conversion, just that the rate is the same. Bloomberg actually agrees with google on this.

![image](https://cloud.githubusercontent.com/assets/13737234/9341841/8d758ed0-45f8-11e5-8ad2-4b7aedfb28b6.png)

Anyway I'm not gonna bother you guys with this any further.

**Comment by ArunGupta25** [comment]
Are you sure it converted it to 267.59 euro? I just checked our currency conversion function and it gave me 271.96 euro for 300 USD.

**Comment by albo-cadd9** [comment]
you're right, I fucked up my math, thanks for your patience (maybe that's why I'm a designer and not a coder). Anyway, glad to see that now it's working well!

Any chance of putting it as a global variable instead of selecting it every time for every item? Would be handy also for people who already have a big wardrobe and want to enable the additional currency.

**Comment by albo-cadd9** [comment]
I'm trying to edit my old items (listings made before the change went live) to use € but they keep reverting to $.

**Comment by albo-cadd9** [comment]
also buyers can't make an offer in currencies other than dollars
